src/pymedphys/dicom/_level2/dicom_anonymise.py

```diff
@@ -31,73 +31,85 @@
+                        "CurveDate",
+                        "CurveTime",
+                        "Date",
+                        "DateTime",
+                        "EthnicGroup",
+                        "InstanceCreationDate",
+                        "InstanceCreationTime",
+                        "InstanceCreatorUID",
+                        "InstitutionAddress",
+                        "InstitutionalDepartmentName",
+                        "InstitutionName",
+                        "IssuerOfPatientID",
+                        "NameOfPhysiciansReadingStudy",
+                        "OperatorsName",
+                        "OtherPatientIDs",
+                        "OtherPatientNames",
+                        "OverlayDate",
+                        "OverlayTime",
+                        "PatientAddress",
+                        "PatientAge",
+                        "PatientBirthDate",
+                        "PatientBirthName",
+                        "PatientBirthTime",
+                        "PatientID",
+                        "PatientInstitutionResidence",
+                        "PatientMotherBirthName",
+                        "PatientName",
+                        "PatientSex",
+                        "PatientTelephoneNumbers",
+                        "PerformingPhysicianIdentificationSequence",
+                        "PerformingPhysicianName",
+                        "PersonName",
+                        "PhysiciansOfRecord",
+                        "PhysiciansOfRecordIdentificationSequence",
+                        "PhysiciansReadingStudyIdentificationSequence",
+                        "ReferringPhysicianAddress",
+                        "ReferringPhysicianIdentificationSequence",
+                        "ReferringPhysicianName",
+                        "ReferringPhysicianTelephoneNumbers",
+                        "RegionOfResidence",
+                        "ReviewerName",
+                        "SecondaryReviewerName",
+                        "SeriesDate",
+                        "SeriesTime",
+                        "StationName",
+                        "StudyDate",
+                        "StudyID",
+                        "StudyTime",
+                        "Time",
+                        "VerifyingObserverName")
+
+
+VR_ANONYMOUS_REPLACEMENT_VALUE_DICT = {'AS': "100Y",
+                                       'CS': "ANON",
+                                       'DA': "20190303",
+                                       'DT': "20190303000900.000000",
+                                       'LO': "Anonymous",
+                                       'PN': "Anonymous",
+                                       'SH': "Anonymous",
+                                       'SQ': ["Anonymous"],
+                                       'ST': "Anonymous",
+                                       'TM': "000900.000000",
+                                       'UI': "12345678"}
```

Comment:
Very neat approach :). Hangover can't have been that debilitating :)

Reply:
Hahaha, thanks :)